r2r/main/app.py

```diff
@@ -69,6 +73,32 @@ def create_app(
     if not upload_path.exists():
         upload_path.mkdir()
 
+    @app.post("/process_url")
```

Comment:
Is there any way to make this optional through configuration? I think this is something which we may want to toggle on / off.